sdk/metricsadvisor/Azure.AI.MetricsAdvisor/perf/Tests/GetAnomaliesForAlertTest.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.AI.MetricsAdvisor.Perf
+{
+    public class GetAnomaliesForAlertTest : MetricsAdvisorTest<MetricsAdvisorPerfOptions>
```

Comment:
It is better to not suffix the name of the actual tests with `Test`.
Then, the command-line reads better because you pass the test name as the operation itself.

Also, consider marking the classes `sealed` unless there is a reason not to.